### PR TITLE
Fix/icon colour change

### PR DESCRIPTION
**Pull request by ponrajaprabhusk**
### Functionality:
Changed the color of Priority and Difficulty icons

### Solution:
Changed the icon colors of by changing the color code

### Risk level:
- [ ] high 
- [ ] medium
- [X] low

### How to test:
Add task for each Priority and difficulty and check whether the correct colors are displayed

### CODE REVIEW

**Review by Pendyala-thanvi — COMMENTED** [review]
Revert changes in deleteFilesInOrg file. The whole code seems to be deleted.

File: functions/model/librarian/tark/deleteFilesInOrg.js [hunk 1]
```diff
@@ -1,43 +0,0 @@
-/* eslint-disable linebreak-style */
-/* eslint-disable max-len */
-/* eslint-disable no-trailing-spaces */
-/* eslint-disable object-curly-spacing */
-/* eslint-disable no-unused-vars */
```
Comment (viv9k) [inline functions/model/librarian/tark/deleteFilesInOrg.js:5]: why did you removed this file

**Review by viv9k — CHANGES_REQUESTED** [review]
One file is removed please revert it.

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**